app/presenters/specialist_document_presenter.rb

```diff
@@ -89,32 +91,52 @@ def facets_with_values
     end
   end
 
+  def friendly_facet_date(dates)
+    dates.map { |date| display_date(date) }
+  end
+
   def friendly_facet_text(facet, values)
     if facet['allowed_values'] && facet['allowed_values'].any?
-      friendly_facet_allowed_values(facet, values)
+      check_allowed_values(facet, values)
+      facet_blocks(facet, values)
     else
       values
     end
   end
 
-  def friendly_facet_allowed_values(facet, values)
-    # TODO: Shout loudly if a value isn't one of the allowed ones
-    facet['allowed_values'].select { |v| values.include?(v['value']) }.map do |allowed_value|
-      facet['filterable'] ? filterable_facet_link(facet, allowed_value) : allowed_value['label']
+  def check_allowed_values(facet, values)
+    allowed_values = facet['allowed_values'].map { |av| av["value"] }
+    not_allowed = "facet value not in list of allowed values"
+
+    values.each do |v|
+      Airbrake.notify(not_allowed) unless allowed_values.include?(v)
```

Comment:
We should include the not allowed value in the Airbrake notification for easier debugging.